tools/maplint/lints/table_pixel_shifts.yml

```
@@ -0,0 +1,4 @@
+/obj/structure/table:
+  banned_variables:
+    pixel_x:
+    pixel_y:
```

```suggestion
  - pixel_x
  - pixel_y
```

This is only syntactic, but name: should be used for when we are defining something inside of it:

If we want to be more specific, we can write out the specific variables we want to ban.

```yml
/mob/dog
  banned_variables:
  - species # Don't var-edit species, use the subtypes
```

We can also explicitly create allowlists and denylists of values through `allow` and `deny`. For example, if we want to make sure we're not creating invalid bowls for animals, we can write:

```yml
/obj/bowl/dog:
  banned_variables:
    species:
      # If we specify a species, it's gotta be a dog
      allow: ["beagle", "corgi", "pomeranian"]

/obj/bowl/humans:
  banned_variables:
    species:
      # We're civilized, we don't want to eat from the same bowl that's var-edited for animals
      deny: ["cats", "dogs"]
```